lib/ansible/modules/cloud/amazon/ec2_instance.py

```diff
@@ -644,16 +647,16 @@
             returned: always
             type: complex
```

Comment:
Shouldn't this be: type: list, elements: dict?

Reply:
Yes, good catch! Thanks!